jobs/bbr-postgres-db/templates/restore.sh.erb

```diff
@@ -41,6 +41,9 @@ for dbname in ${DATABASES[@]}; do
       --use-list=${TMP_LIST_FILE} \
       --if-exists \
       --clean \
+      <% if p("postgres.single-transaction") %>
```

Comment:
@ramonskie Thanks for the PR. The empty line left behind by the tag after the erb processing causes the `      "${BBR_ARTIFACT_FILE_PATH}"` line not to be concatenated to the restore command. 
I suggest to create a variable that contains the `--single-transaction` only if the new property is specified; then you can unconditionally use the variable with the continuation character in the restore command.

Reply:
changed the \ so that even when false an empty line will be added to the command.

Reply:
It seems not to work when the property is true:
<img width="273" alt="image" src="https://user-images.githubusercontent.com/5594943/73651467-c7352e00-4684-11ea-859b-92a3cdd2b0e2.png">

Reply:
followed your advise and putted it in to a variable.
otherwise there would be to much empty line with `\` in the command